README.md

```diff
@@ -214,7 +221,14 @@ configured explicitly when invoking Docker.
 Applications that run under Docker are usually easy to run on
 OpenShift.  There will be configuration changes.  OpenShift
 configuration is beyond the scope of this readme.  The *runDocker.sh*
-script is a model for what OpenShift needs to be configured to supply.
+script is a model for what the OpenShift environment needs to supply for correct configuration.
+
+The version of kartograafr to run is specified in the OpenShift build configuration.  
+It should point to a git tag for testing and production instances.
```

Comment:
You should explain in which configuration the tag is specified.  I saw a couple of config-like things in OS and I happened to find the right one by trial and error.

Also, rather than giving a specific version numbered tag, could OS build based on:
* Whichever tag is the latest?
* Always build a tag called "production" (without a version number)?

And is there any reason those should be avoided?